Algorithm.Framework/Selection/ManualUniverse.cs

```diff
@@ -28,6 +29,7 @@ namespace QuantConnect.Algorithm.Framework.Selection
     /// </summary>
     public class ManualUniverse : UserDefinedUniverse
     {
+        /// <inheritdoc />
```

Comment:
I don't think our doc generator supports the `inheritdoc` XML tag yet

Reply:
Good catch, I'll remove the `inheritdoc` XML tag